.rhcicd/clowdapp-engine.yaml

```diff
@@ -221,6 +226,15 @@ objects:
                   - type: Pods
                     value: 2
                     periodSeconds: 30
+- apiVersion: keda.sh/v1alpha1
+  kind: TriggerAuthentication
+  metadata:
+    name: cma-prometheus-trigger-auth
+  spec:
+    secretTargetRef:
+    - parameter: bearerToken
+      name: notifications-${ENV_NAME}-app-sre-observability-per-cluster-notifs-prom
```

Comment:
Will this work in ephemeral?

Reply:
Replied on [Slack](https://redhat-internal.slack.com/archives/C024CDDPRHD/p1757081314697469?thread_ts=1757024773.524909&cid=C024CDDPRHD). Same as previous answer.